resources/knative-eventing/charts/knative-eventing/templates/pre-upgrade-job.yaml

```diff
@@ -58,6 +71,339 @@ data:
       channels.eventing.knative.dev --ignore-not-found
     #### deploy/event-bus-subscription-controller adds/removes a finalizer to knative subscriptions hence it needs to be deleted after knative subscriptions are deleted
     kubectl delete -n kyma-system deploy/event-bus-subscription-controller --ignore-not-found
+
+    #### Create newly introduced CRDs in 0.10 knative-eventing charts
+    cat << EOF | kubectl apply -f -
+    apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Can't we just make this part of the eventing YAML? All missing objects should be automatically created during update.